### PR TITLE
Fix part #703: Domain Subtopic Thumbnail Introduction

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR focuses on introducing thumbnail in subtopics in Domain layer.

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@veena14cs title should not include `Low-fi`, low-fi specifically means that you are working on app module. Better write it like this:

`Domain Subtopic Thumbnail Introduction`

File: domain/src/main/java/org/oppia/domain/topic/TopicController.kt [hunk 1]
```diff
@@ -37,7 +39,8 @@ const val FRACTIONS_SKILL_ID_1 = "UxTGIJqaHMLa"
 const val FRACTIONS_SKILL_ID_2 = "B39yK4cbHZYI"
 const val FRACTIONS_SUBTOPIC_ID_1 = "1"
 const val FRACTIONS_SUBTOPIC_ID_2 = "2"
-const val FRACTIONS_SUBTOPIC_ID_3 = "3"
+const val FRACTIONS_SUBTOPIC_ID_3= "3"
```
Comment (rt4914) [inline domain/src/main/java/org/oppia/domain/topic/TopicController.kt]: Nit spacing issue
`const val FRACTIONS_SUBTOPIC_ID_3 = "3"`

File: domain/src/test/java/org/oppia/domain/topic/TopicControllerTest.kt [hunk 2]
```diff
@@ -634,6 +635,25 @@ class TopicControllerTest {
...
+    val topic = topicLiveData.value!!.getOrThrow()
+    assertThat(topic.subtopicList.get(0).subtopicThumbnail.thumbnailGraphic).isEqualTo(SubtopicThumbnailGraphic.WHAT_IS_A_FRACTION)
+  }
+
+  @Test
+  fun testRetrieveSubtopicTopic_validSubtopic_subtopicsHaveNoThumbnailUrls() {
+    val topicLiveData = topicController.getTopic(FRACTIONS_TOPIC_ID)
+
+    // This test is intentionally verifying that there are no thumbnails for subtopics, since there are not yet any to
+    // populate.
```
Comment (rt4914) [inline domain/src/test/java/org/oppia/domain/topic/TopicControllerTest.kt]: You can remove this comment as your test name is self explanatory. Also, this is good that you are writing this test case.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: domain/src/main/java/org/oppia/domain/topic/TopicController.kt (hunk 1 above)
Comment (veena14cs) [reply domain/src/main/java/org/oppia/domain/topic/TopicController.kt]: Done.

File: domain/src/test/java/org/oppia/domain/topic/TopicControllerTest.kt (hunk 2 above)
Comment (veena14cs) [reply domain/src/test/java/org/oppia/domain/topic/TopicControllerTest.kt]: Done.

**Comment by rt4914** [comment]
@veena14cs there are merge conflicts in this file.

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM

**Comment by rt4914** [comment]
@veena14cs Can you resolve these merge conflicts and merge this PR.

**Pull request merged**